proto/api/v1/search_service.proto

```diff
@@ -65,9 +65,10 @@ enum SearchCategory {
     REPORT_SNAPSHOT           = 53;
     COMPLIANCE_INTEGRATIONS   = 54;
     COMPLIANCE_SCAN_SETTINGS  = 55;
-    COMPLIANCE_SCAN   = 56;
+    COMPLIANCE_SCAN           = 56;
     COMPLIANCE_CHECK_RESULTS  = 57;
     BLOB                      = 58;
+    PRODUCT_USAGE             = 59;
```

Comment:
I don't think you need this. IIRC, this is only if you want this tag to be searchable in the search bar in UI.

`search:` tag in proto should be enough for the store to create an index.

Reply:
The category is used in `store/gen.go` to tell the generator to generate `GetByQuery()`, IIUC.